### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by kbartush**
Additions to the mazzaroth-cli readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,155 @@ The CLI can be installed by using npm.
...
-Requests/Responses defined [here](https://github.com/kochavalabs/mazzaroth-xdr/blob/develop/idl/rpc.x).
-For more context, an example of using the CLI with an actual contract can be
-found [here](https://github.com/kochavalabs/full-contract-example).
+Below is the list of commands provided by mazzaroth-cli. The more complex
+commands will be explained in further detail below.
+
+| Command | Description | Example |
+| ------- | ----------- | ------- |
+| transaction-call | Submits a call transaction to a mazzaroth node. Arguments are XDR formatted as  base64 encoded strings. | mazzaroth-cli transaction-call my_func -a 9uZz -a f1zsfABG7J |
+| readonly-call | Submits a readonly call transaction to a mazzaroth node. Arguments are XDR formatted as  base64 encoded strings. | mazzaroth-cli readonly-call my_func -a 9uZz -a f1zsfABG7J |
```
Comment (jeff1010322) [inline README.md:26]: Extra space between "as base64" in both of these lines.

File: README.md [hunk 2]
```diff
@@ -15,13 +15,155 @@ The CLI can be installed by using npm.
...
+| contract-update | Submits an update transaction to a mazzaroth node. The format of the argument is a path to a file containing contract wasm bytes. | mazzaroth-cli contract-update ./test/data/hello_world.wasm |
+| permission-update | Submits a permission transaction to a mazzaroth node that allows another account to act on your behalf. The argument to this call is the public key of the account you would like to grant access to. | mazzaroth-cli permission-update 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| transaction-lookup |  Looks up the current status and results of a transaction by ID. Argument is a transaction ID (256 bit hex value). | mazzaroth-cli transaction-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
```
Comment (jeff1010322) [inline README.md:29]: Extra space before "Looks" here.

File: README.md [hunk 3]
```diff
@@ -15,13 +15,155 @@ The CLI can be installed by using npm.
...
+| block-lookup | Looks up a Block using either a block ID as hex or block Number. | mazzaroth-cli block-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| block-header-lookup | Looks up a Block Header using either a block ID as hex or block Number. | mazzaroth-cli block-header-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| receipt-lookup | Looks up a transaction receipt. Argument is a transaction ID (256 bit hex value). | mazzaroth-cli receipt-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| nonce-lookup | Looks up the current nonce for an account, Argument is an account ID (256 bit hex value). | mazzaroth-cli nonce-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| account-lookup | Looks up the current information for an account, Argument is an account ID (256 bit hex value). | mazzaroth-cli account-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| channel-lookup | Looks up the current information for a channel, Argument is what specifically to lookup about the channel. Current options: config/contract | mazzaroth-cli channel-lookup config |
```
Comment (jeff1010322) [inline README.md:35]: Should the commas here be periods?   Used a period in the receipt-lookup description.

File: README.md [hunk 4]
```diff
@@ -15,13 +15,155 @@ The CLI can be installed by using npm.
...
+#       "channelID":"0000000000000000000000000000000000000000000000000000000000000000",
+#       "nonce":"0",
+#       "category":{
+#          "enum":0,
+#          "value":""
+#       }
+#    }
+# }
+
+# Translate a ReceitSubscriptionResult from base64
```
Comment (jeff1010322) [inline README.md:155]: Misspelled "ReceiptSubscriptionResult"

File: README.md [hunk 5]
```diff
@@ -15,13 +15,155 @@ The CLI can be installed by using npm.
...
+| block-lookup | Looks up a Block using either a block ID as hex or block Number. | mazzaroth-cli block-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| block-header-lookup | Looks up a Block Header using either a block ID as hex or block Number. | mazzaroth-cli block-header-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| receipt-lookup | Looks up a transaction receipt. Argument is a transaction ID (256 bit hex value). | mazzaroth-cli receipt-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| nonce-lookup | Looks up the current nonce for an account. Argument is an account ID (256 bit hex value). | mazzaroth-cli nonce-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| account-lookup | Looks up the current information for an account, Argument is an account ID (256 bit hex value). | mazzaroth-cli account-lookup 3a547668e859fb7b112a1e2dd7efcb739176ab8cfd1d9f224847fce362ebd99c |
+| channel-lookup | Looks up the current information for a channel, Argument is what specifically to lookup about the channel. Current options: config/contract | mazzaroth-cli channel-lookup config |
```
Comment (jeff1010322) [inline README.md:35]: These 2 lines still have the commas.

Comment (kbartush) [reply README.md:35]: Guess I copied it a few times.

One sec.

**Pull request merged**